rewards/boost/calc_boost.py

```diff
@@ -44,8 +54,9 @@ def calc_stake_ratio(address: str, boost_bals: BoostBalances) -> Decimal:
     native_balance = Decimal(boost_bals.native.get(address, 0))
     non_native_balance = Decimal(boost_bals.non_native.get(address, 0))
     bvecvx_balance = Decimal(boost_bals.bvecvx.get(address, 0))
-    native_balance += calc_bvecvx_native_balance(
-        native_balance, bvecvx_balance)
+    digg_balance = Decimal(boost_bals.digg.get(address, 0))
+    native_balance += calc_bvecvx_native_balance(native_balance, bvecvx_balance)
+    native_balance += calc_digg_native_balance(native_balance, digg_balance)
```

Comment:
This should be switched.  Per the BIP, `DIGG token contribution to boost will be limited to the total USD value of an addresses BADGER, remBADGER and BADGER NFT` CVX balances should not count towards the limitation